src/components/TriggerEditor/index.tsx

```diff
@@ -460,6 +495,36 @@ export default ({
             realm={realm}
             validationErrors={actionValidationErrors}
           />
+          <Form>
+            <Form.Row className="mb-2">
+              <Col sm={12}>
+                <Form.Group controlId="triggerPolicyName">
+                  <Form.Label>Policy name</Form.Label>
+                  <Form.Control
+                    as="select"
+                    name="triggerPolicyName"
+                    disabled={isReadOnly || isLoadingPoliciesName}
+                    value={_.get(triggerDraft, 'policy')}
+                    onChange={handleTriggerPolicyNameChange}
+                    isInvalid={_.get(triggerValidationErrors, 'policy') != null}
+                  >
+                    <option value="" disabled>
+                      Choose available delivery trigger policy
+                    </option>
+                    <option value="">No custom policy should be used</option>
+                    {policiesName.map((policy) => (
+                      <option key={policy} value={policy}>
+                        {policy}
+                      </option>
+                    ))}
+                  </Form.Control>
+                  <Form.Control.Feedback type="invalid">
+                    {_.get(triggerValidationErrors, 'policy')}
+                  </Form.Control.Feedback>
+                </Form.Group>
```

Comment:
Nice work here, just a few notes. Upon internal discussion with the Cloud team, we feel that the best phrasing here would be:
- `Trigger delivery policy` for the form label
- `Choose available trigger delivery policy` for the prompt option
- `Use default policy` for the default blank option